README.md

```diff
@@ -54,7 +54,7 @@ to configure and install Rpanion-server with all required dependencies. Note thi
 an initial Wifi hotspot.
 
 ```
-cd ./deploy && ./RasPi2-3-4-5-deploy.sh
+cd .~/Rpanion-server/deploy && ./RasPi2-3-4-5-deploy.sh
```

Comment:
Remove this, as it might confuse people who don't install Rpanion-server in ``~/``

Reply:
Done